assets/_scss/vkExUnit_admin.scss

```
@@ -204,6 +204,36 @@
 //
 // specificity の方針: すべて class セレクタで揃え、`!important` は使わない。
 // shorthand `margin: 0 0 0.3em` ではなく `margin-top` / `margin-bottom` を個別に書く。
+// Customizer「ページトップへ戻るボタン」セクション内の見出し余白拡大。
+// issue #1368: 親グループ見出し「Page top button image」(h2.admin-custom-h2) と
+// 子グループ見出し「Image size」(h3.admin-custom-h3) の上下余白がデフォルトのままだと
+// 隣接 control との間が詰まりすぎ、見出しと中身が一塊に見えない（情報グルーピングが弱い）。
+// design-rules.md「余白」「情報グループ内の余白 < 情報グループ間の余白」を踏まえ、
+// 当該セクション内に限り margin を 1.4 倍に拡大する（指定範囲 1.3〜1.5 の中央）。
+//
```

Stylelint (17.12.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)